service/sg-windows/sg-service/sg-service.go

```diff
@@ -76,26 +76,36 @@ func main() {
 
 	var exePath string
 	var configPath string
+	var stderrPath string
 
 	switch len(os.Args) {
 	case 2:
 		exePath = "C:\\Program Files (x86)\\Couchbase\\sync_gateway.exe" // Uses default binary image path
-		svcConfig.Arguments = []string{"start"}                          // Uses the default config
+		stderrPath = "C:\\Program Files (x86)\\Couchbase\\var\\lib\\couchbase\\logs\\sync_gateway_error.log" // Uses default stderr path
+		svcConfig.Arguments = []string{"start", stderrPath}                          // Uses the default config
 	case 3:
 		exePath = "C:\\Program Files (x86)\\Couchbase\\sync_gateway.exe" // Uses default binary image path
 		configPath = os.Args[2]                                          // Uses custom config
-		svcConfig.Arguments = []string{"start", configPath}
+		stderrPath = "C:\\Program Files (x86)\\Couchbase\\var\\lib\\couchbase\\logs\\sync_gateway_error.log" // Uses default stderr path
+		svcConfig.Arguments = []string{"start", configPath, stderrPath}
 	case 4:
 		exePath = os.Args[2]    // Uses custom binary image path
 		configPath = os.Args[3] // Uses custom config
-		svcConfig.Arguments = []string{"start", exePath, configPath}
+		stderrPath = "C:\\Program Files (x86)\\Couchbase\\var\\lib\\couchbase\\logs\\sync_gateway_error.log" // Uses default stderr path
+		svcConfig.Arguments = []string{"start", exePath, configPath, stderrPath}
+	case 5:
+		exePath = os.Args[2]    // Uses custom binary image path
+		configPath = os.Args[3] // Uses custom config
+		stderrPath = os.Args[4]
```

Comment:
If these are going to be using positional (as opposed to named) arguments, is the usage documented anywhere?  

Or is this usage always strictly internal by a service script?  In that case, maybe it's less important to document.  Do Windows users typically start Sync Gateway on the Command Prompt?